### PR TITLE
[FIX] Hours, minutes, and seconds are always displayed as plural for next and prev commands

### LINKED ISSUE #67: Hours, minutes, and seconds are always displayed as plural for next and prev commands
## Bug Description

When using the `::hydrate next` and `::hydrate prev` commands, the output text for hours, minutes, and seconds is always displayed in plural even when it shouldn't be. As a result, it can sometimes display incorrectly like the following: `1 hours 1 minutes 1 seconds`.

## Steps to Reproduce

1. Set the hydrate interval to 2 minutes
2. Run the chat command `::hydrate next`
3. Observe that the output is `0 hours 1 minutes _ seconds`

## Expected Behavior

It is expected for the unit of time to be properly set to plural or singular when displayed via the `::hydrate next` and `::hydrate prev` commands.
For example, `1 hour 24 minutes 1 second` or `0 hours 1 minute 59 seconds`.

## Screenshots

Screen Shot 2021-07-15 at 1 23 34 AM

## Environment

Operating System: macOS Big Sur
RuneLite Version: 1.7.15

### PR DESCRIPTION
**Pull request by medkhabt**
## Associated Issue

Closes #67

## Implemented Solution
I added a method to take in charge the string formatting, and i used it in the methods that have the logic behind "::hydrate next" and "::hydrate  prev"

## Testing Details
By testing 3 cases: 1 , 0 and a different number than these two.
I couldn't test that in rs, as i don't play the game and i don't know how to set it up.

### CODE REVIEW

File: src/main/java/com/hydratereminder/HydrateReminderPlugin.java [hunk 1]
```diff
@@ -299,11 +298,27 @@ private void handleHydratePrevCommand()
...
+	 * @param hours
+	 * @param minutes
+	 * @param seconds
+	 * @return Time in a string format
+	 * @since 1.1.1
+	 */
+	String timeDisplay(int hours, int minutes, int seconds) {
+		String hoursRepresentation = hours != 1 ? hours + " hours" : hours + " hour";
+		String minutesRepresentation = minutes != 1 ? minutes + " minutes" : minutes + " minute";
+		String secondesRepresentation = seconds != 1 ? seconds + " secondes" : seconds + " second";
```
Comment (jmakhack) [inline src/main/java/com/hydratereminder/HydrateReminderPlugin.java:316]: Can you correct the spelling of `secondes` to `seconds`?
Looks like the unit tests will have to be updated as well.

**Review by jmakhack — CHANGES_REQUESTED** [review]
Thanks for your contribution to this project. The code looks good to me.
Just requesting a small spelling change then this pull request will be ready to merge.

**Comment by jmakhack** [comment]
@all-contributors please add @medkhabt for code and tests

File: src/main/java/com/hydratereminder/HydrateReminderPlugin.java [hunk 2]
```diff
@@ -299,11 +298,27 @@ private void handleHydratePrevCommand()
...
 
+	/**
+	 * <p>Handle the String formatting of the specified time.</p>
+	 * @param hours
+	 * @param minutes
+	 * @param seconds
+	 * @return Time in a string format
+	 * @since 1.1.1
+	 */
+	String timeDisplay(int hours, int minutes, int seconds) {
```
Comment (jmakhack) [inline src/main/java/com/hydratereminder/HydrateReminderPlugin.java:313]: Can you also take a look into fixing this code quality alert?
Looks like it expects explicit scoping to be done here.

https://app.codacy.com/gh/jmakhack/hydrate-reminder/pullRequest?prid=7761949

File: src/main/java/com/hydratereminder/HydrateReminderPlugin.java (hunk 1 above)
Comment (medkhabt) [reply src/main/java/com/hydratereminder/HydrateReminderPlugin.java:316]: Ok i'll do that.

File: src/main/java/com/hydratereminder/HydrateReminderPlugin.java (hunk 2 above)
Comment (medkhabt) [reply src/main/java/com/hydratereminder/HydrateReminderPlugin.java:313]: Ok i'll make it public.

**Review by jmakhack — APPROVED** [review]
Approving your changes and merging it in.
Also tested the change in-game and it works as expected.

Screen Shot 2021-07-16 at 1 41 33 AM

**Pull request merged**